applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -236,28 +236,28 @@ static void stream_started_cb(struct bt_audio_stream *stream)
 {
 	int ret;
 
-	LOG_INF("Audio stream %p started", (void *)stream);
+	LOG_INF("Stream started");
 
 	ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_STREAMING);
 	ERR_CHK(ret);
 }
 
 static void stream_metadata_updated_cb(struct bt_audio_stream *stream)
 {
-	LOG_INF("Audio Stream %p metadata updated", (void *)stream);
+	LOG_DBG("Audio Stream %p metadata updated", (void *)stream);
 }
 
 static void stream_disabled_cb(struct bt_audio_stream *stream)
 {
-	LOG_INF("Audio Stream %p disabled", (void *)stream);
+	LOG_DBG("Audio Stream %p disabled", (void *)stream);
 }
 
 static void stream_stopped_cb(struct bt_audio_stream *stream)
 {
 	int ret;
 	uint8_t channel_index;
 
-	LOG_INF("Audio Stream %p stopped", (void *)stream);
+	LOG_INF("Stream stopped");
```

Comment:
Same as above, since gateway needs to handle two connections, proabably we still need to add something for identification.